airflow/providers/microsoft/winrm/operators/winrm.py

```diff
@@ -56,6 +56,7 @@ class WinRMOperator(BaseOperator):
     """
 
     template_fields = ('command',)
+    template_fields_renderers = {"command": "bash"}
```

Comment:
Since https://pygments.org/docs/lexers/#pygments.lexers.shell.PowerShellLexer exists should we not use it?

Reply:
Right, I should have thought of it 😅